src/sentry/tasks/beacon.py

```diff
@@ -68,6 +75,32 @@ def get_events_24h() -> int:
     return sum_events
 
 
+def get_category_event_count_24h() -> int:
+    from sentry.models.organization import Organization
+
+    organization_ids = list(Organization.objects.all().values_list("id", flat=True))
+    event_categories_count = {"error": 0, "replay": 0, "transaction": 0, "profile": 0, "monitor": 0}
+    for organization_id in organization_ids:
+        # Utilize the outcomes dataset to send snql queries for event stats
+        query = QueryDefinition(
+            fields=["sum(quantity)"],
+            organization_id=organization_id,
+            stats_period="24h",
+            group_by=["category"],
+            outcome=["accepted"],
+        )
+        tenant_ids = {"organization_id": organization_id}
+        with sentry_sdk.start_span(op="outcomes.endpoint", description="run_outcomes_query"):
```

Comment:
What is the utility of having this span here, if only self-hosted users, who don't have sentry installed, will use this endpoint?

Reply:
guess this doesn't have much utility, removed